app_dart/bin/server.dart

```diff
@@ -82,29 +93,135 @@ Future<void> main() async {
         authProvider,
         scheduler,
       ),
+
+      /// Updates cocoon task status.
+      ///
+      /// This API updates task status in datastore.
+      ///
+      /// POST: /api-update-status
+      ///
+      /// Parameters:
+      ///   CommitBranch: (string in body). Branch of commit.
+      ///   CommitSha: (string in body). Sha of commit.
+      ///   BuilderName: (string in body). Name of the luci builder.
+      ///   NewStatus: (string in body) required. Status of the task.
+      ///   ResultData: (string in body) optional. Benchmark data.
+      ///   BenchmarkScoreKeys: (string in body) optional. Benchmark data.
+      ///
+      /// Response: Status 200 OK
+      ///
+      /// Requires authentication: Status: 401 Unauthorized
       '/api/update-task-status': UpdateTaskStatus(config, swarmingAuthProvider),
       '/api/vacuum-github-commits': VacuumGithubCommits(
         config,
         authProvider,
         scheduler: scheduler,
       ),
+
+      /// Returns status of the tree.
+      ///
+      /// Returns serialized proto with enum representing the
+      /// status of the tree and list of offending tasks.
+      ///
+      /// GET: /api/public/build-status
+      ///
+      /// Parameters:
+      ///   branch: (string in query) default: 'master'. Name of the repo branch.
+      ///
+      /// Response: Status 200 OK
```

Comment:
Can you link to the proto instead? For example, 2 doesn't make sense, but with context it maps to a logical enum

Reply:
done

Reply:
Is this what you meant?

Reply:
Breadcrumbs are dart types in brackets

For example:
```dart
/// Returns [BuildStatusResponse]
```